### ISSUE TITLE
Error when using the script

### ISSUE THREAD

**Issue by marcostudios** [SlimShadyIAm/PreMiD-linux-scripts#2]
As https://github.com/PreMiD/PreMiD/issues/223 the same error appears at line 40 "npm run pkglinux". 

The error is "npm ERR! missing script: pkglinux"

**Comment by SlimShadyIAm** [comment]
Yeah, PreMiD was updated to v2 which changed the way you package the app. I have to update the script. Will do as soon as possible, probably this evening.

**Comment by JohnTheCoolingFan** [comment]
"Probably this evening"
*10 days ago*

**Comment by SlimShadyIAm** [comment]
I can’t do anything until Timeraa writes the package script for Linux he
said he would
-- 
Aamir Farooq
(aamircute3@gmail.com)
